wporg-two-factor.php

```diff
@@ -35,9 +35,14 @@ function two_factor_providers( array $providers ) : array {
 	$desired_providers = array(
 		'Two_Factor_WebAuthn'     => '',
 		'Two_Factor_Totp'         => '',
-		'Two_Factor_Backup_Codes' => '',
 	);
 
+	// Check current user has a primary method setup, before allowing backup codes
+	$user_providers = Two_Factor_Core::get_enabled_providers_for_user();
```

Comment:
Should we pass `bbp_get_displayed_user_id()` to the function, for the the situations where an admin is editing another user?